### DISCUSSION TITLE
Auto mod replay username is inconsistent between rulesets.

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#21081] [Q&A] [answered]
Visible in-game with leaderboards enabled

Where do these names come from?
Are they going to stay like this?

Ruleset | osu! | osu!taiko | osu!catch | osu!mania
--- | --- | --- | --- | --- 
Screenshot | ![image](https://user-images.githubusercontent.com/102317848/199613187-fd8bb719-56db-44a1-b80a-77e67e061dc4.png) | ![image](https://user-images.githubusercontent.com/102317848/199613220-065fbd61-4a87-4105-a220-662e4924c45c.png) | ![image](https://user-images.githubusercontent.com/102317848/199613241-22597b0e-1599-4b87-bab5-3562b9efb3ad.png) | ![image](https://user-images.githubusercontent.com/102317848/199613311-96320f1a-8a58-4212-bc06-ace4679ca98b.png) 
Name | Autoplay | mekkadosu! | osu!salad | osu!topus

**Comment by mk56-spn** [comment, marked as answer]
https://www.reddit.com/r/osugame/comments/5rbpqq/til_that_the_auto_mod_has_different_name/

They seem to come from stable

**Reply by peppy** [reply]
And is intentional.
